src/context/AppProvider/AppContext.tsx

```diff
@@ -192,6 +192,8 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
           case osmosisChainId:
           case avalancheChainId:
             ;(async () => {
+              dispatch(zapper.endpoints.getZapperAppsBalancesOutput.initiate())
```

Comment:
we still can't move this up to 183?

Reply:
https://github.com/shapeshift/web/pull/4504/commits/6e9a2e096bec2b52fdea6aba12b7958653b71007 absolootely can